portfolio/src/main/webapp/index.html

```diff
@@ -8,7 +8,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Bootstrap 4 Portfolio/Resume Theme for Developers">
     <meta name="author" content="Xiaoying Riley at 3rd Wave Media">    
-    <link rel="shortcut icon" href="favicon.ico">  
```

Comment:
Did you used to have one or was it never there?

Reply:
This came with the template and has been causing the console to log errors. Seems like it is not doing anything but errors persist even when I removed it so not need to do a deeper dive

Reply:
Yeah browsers look for favicons even id you don't indicate there is one in the HTML page.